cluster-up/cluster/kind/common.sh

```diff
@@ -324,7 +324,7 @@ function down() {
     $KIND delete cluster --name=${CLUSTER_NAME}
     rm -f ${KUBEVIRTCI_CONFIG_PATH}/$KUBEVIRT_PROVIDER/kind.yaml
 
-    if [ "$ETCD_DATA_DIR" != none ] && [ "$BASE_ETCD_DATA_DIR" != none ];then
+    if [ -d "$BASE_ETCD_DATA_DIR" ];then
```

Comment:
This is going to be true always

Reply:
Right, I pushed new commit to run teardown only on local environments